modules/docs/arrow-docs/docs/docs/quickstart/projects/README.md

```diff
@@ -51,3 +51,5 @@ Note: This section keeps on growing! Keep an eye on it from time to time.
 [pedrovgs/KotlinKatas](https://github.com/pedrovgs/KotlinKatas): Kotlin training repository used to learn Kotlin and Functional Programming by solving some common katas using just purely functional programming powered by Arrow.
 
 [arrow-kt/frdomain.kt](https://github.com/arrow-kt/frdomain.kt): Arrow port of the additional code samples (Chapter 6) from [Functional and Reactive Domain Modeling](https://www.manning.com/books/functional-and-reactive-domain-modeling).
+
+[adomokos/arrow-examples](https://github.com/adomokos/kotlin-sandbox/tree/master/src/test/kotlin/sandbox/arrow): Using [kotlintest](https://github.com/kotlintest/kotlintest) to demonstrate various features of Arrow.
```

Comment:
Hi @adomokos thank you for the contribution!

To follow the other links in the list I'd suggest the following nit:

```suggestion
[adomokos/kotlin-sandbox](https://github.com/adomokos/kotlin-sandbox/): Using [kotlintest](https://github.com/kotlintest/kotlintest) to demonstrate various features of Arrow.
```

Apart from that, looks good to me 👍

Reply:
I use that repo for all my Kotlin learning, I wanted to point the reader to the Arrow examples only. It's listed on the README, I am happy to make this change. Thanks for your comment.